src/sentry/ingest/ingest_consumer.py

```diff
@@ -153,6 +149,21 @@ def _do_process_event(message, projects):
     # which assumes that data passed in is a raw dictionary.
     data = json.loads(payload)
 
+    if killswitch_matches_context(
```

Comment:
I'd suggest we do this check earlier (as early as possible, even if it means omitting things like platform), or we add a new one. 
With the current approach, quite a lot of things can go wrong before we reach this check.

Reply:
ok this means we can basically only use project_id

Reply:
Yes, I think that's fine for now.